bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -97,7 +97,7 @@ <h1 class="large">{{_('Check Your Plugins')}}</h1>
       <div class="version-message-container">
         <h2>{{_('Looks like you’re using an older version of Firefox.')}}</h2>
         <div class="desktop download-button-wrapper">
-          {{ download_firefox(simple=true, check_old_fx=true) }}
+          {{ download_firefox(check_old_fx=true) }}
```

Comment:
Double checked with @schalkneethling on IRC and agrees we should probably remove this. Going to make it so.